Model/Config/Config.php

```diff
@@ -16,6 +16,8 @@ class Config
      */
     const MODULE_NAME = 'Omise_Payment';
 
+    const MEDIA_STORAGE_LOCATION = "/omise/qr/";
```

Comment:
I think we need to remove this line too, right?

Reply:
Removed! @ 7662d2e4552dd18eb2b5afce374d48f5f9c2d376